src/wagtailtrans/managers.py

```diff
@@ -8,6 +10,13 @@ def live(self):
         """Return all the live languages."""
         return self.filter(live=True)
 
-    def default(self):
+    def default(self, site=None):
```

Comment:
I think it's more clear to have a ``default_for_site`` method instead of a default which accepts optional argument. something like:

```
def default(self):
    return self.filter(is_default=True)

def default_for_site(self, site):
    if get_wagtailtrans_setting('LANGUAGES_PER_SITE'):
        return self.filter(site_default_language__site=site)
    return self.default()
```